File: VB/LookUpEditWithCheckboxes/Item.vb
Imports DevExpress.Mvvm

Namespace LookUpEditWithCheckboxes

    Public Class Item
        Inherits BindableBase

        Protected _ID As Integer

        Public Property ID As Integer
            Get
                Return _ID
            End Get

            Set(ByVal value As Integer)
                SetProperty(_ID, value, "ID")
            End Set
        End Property

        Protected _Name As String

        Public Property Name As String
            Get
                Return _Name
            End Get

            Set(ByVal value As String)
                SetProperty(_Name, value, "Name")
            End Set
        End Property
    End Class
End Namespace

File: CS/LookUpEditWithCheckboxes/Item.cs
using DevExpress.Mvvm;

namespace LookUpEditWithCheckboxes {
    public class Item : BindableBase{

        protected int _ID;
        public int ID
        {
            get { return this._ID; }
            set { this.SetProperty(ref this._ID, value, "ID"); }
        }


        protected string _Name;
        public string Name
        {
            get { return this._Name; }
            set { this.SetProperty(ref this._Name, value, "Name"); }
        }
    }
}
